app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageComposerViewModel.kt

```diff
@@ -105,12 +107,19 @@ class MessageComposerViewModel @Inject constructor(
     private val getAssetSizeLimit: GetAssetSizeLimitUseCase,
     private val sendKnockUseCase: SendKnockUseCase,
     private val pingRinger: PingRinger,
-    private val imageUtil: ImageUtil
+    private val imageUtil: ImageUtil,
+    private val fileManger: FileManager
```

Comment:
il faut avoir un bon déjeuner pour ne pas avoir trop de faim pendant la journée 🤓 
```suggestion
    private val fileManager: FileManager
```

Reply:
haha a oui 😋